ui/src/app/lib/atlasmap-data-mapper/models/document.definition.model.ts

```diff
@@ -167,14 +129,24 @@ export class DocumentDefinition {
     }
 
     public getName(includeType: boolean): string {
-        let name: string = this.name;
-        if (ConfigModel.getConfig().showTypes && !this.initCfg.type.isPropertyOrConstant()) {
-            const type: string = this.initCfg.type.isJava() ? ' (Java)' : ' (XML)';
-            name += type;
+        let name: string = this.shortName;
+        if (ConfigModel.getConfig().showTypes && !this.isPropertyOrConstant()) {
+            const type: string = this.type;
```

Comment:
Here, both `name` and `type` are assgined right after being declared. Then you do not need any type annotation at all.